### PR TITLE
added new function getPower

### PR DESCRIPTION
**Pull request by gowthamrao**
(also added missing documentation to prior functions)

### CODE REVIEW

**Comment by gowthamrao** [comment]
related to https://github.com/OHDSI/SkeletonComparativeEffectStudy/issues/27

File: R/DataPulls.R [hunk 1]
```diff
@@ -591,4 +598,170 @@ getCohortMethodResult <- function(connection = NULL,
...
+#'                             data from cohort_method_analysis table in comparative effectiveness data model.
+#' @return                     Tibble. 
+#'
+#' @examples
+#' \dontrun{
+#' connectionDetails <- createConnectionDetails(dbms = "postgresql",
+#'                                            user = "joe",
+#'                                            password = "secret",
+#'                                            server = "myserver")
+#' example1 <- getPower(connection = connectionDetails,
```
Comment (schuemie) [inline R/DataPulls.R:641]: Please add comment indicating the following statement is an example when using data in a database

File: R/DataPulls.R [hunk 2]
```diff
@@ -591,4 +598,170 @@ getCohortMethodResult <- function(connection = NULL,
...
+#'                      databaseSchema = 'database.schema',
+#'                      targetId = 100,
+#'                      comparatorId = 200,
+#'                      outcomeId = 111,
+#'                      databaseId = 'HCUP data',
+#'                      analysisId = 2)
+#' example2 <- getPower(cohortMethodResult = cohortMethodResult,
```
Comment (schuemie) [inline R/DataPulls.R:649]: Add another comment here

File: R/DataPulls.R [hunk 3]
```diff
@@ -591,4 +598,170 @@ getCohortMethodResult <- function(connection = NULL,
...
+    .getcohortMethodAnalysis(
+      connection = connection,
+      databaseSchema = databaseSchema,
+      cohortMethodAnalysis = cohortMethodAnalysis,
+      analysisId = analysisId
+    ) %>%
+    dplyr::select(-definition)
+  
+  #fixed parameters
+  alpha <- 0.05
```
Comment (schuemie) [inline R/DataPulls.R]: Why are these parameters not exposed as function arguments?

**Comment by schuemie** [comment]
Looks good. A few minor suggestions (see my comments).

File: R/DataPulls.R (hunk 3 above)
Comment (gowthamrao) [reply R/DataPulls.R]: Agree - lets parameterize

**Review by schuemie — APPROVED** [review]

**Pull request merged**